Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -316,7 +354,7 @@ define([
             gltf : gltfView,
             cull : false,           // The model is already culled by the 3D tiles
             releaseGltfJson : true, // Models are unique and will not benefit from caching so save memory
-            basePath : getBaseUri(this._url, true),
+            basePath : getAbsoluteUri(getBaseUri(this._url, true)),
```

Comment:
This change here was added because tiles pointing to the same resources might have different paths due to `../../` and other path separators. In effect this normalizes the uri.